src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -87,6 +87,7 @@ static void Main(string[] args)
             return AssertCodeCleanupResult(expected, code,
                 (CodeCleanupOptions.AreCodeCleanupRulesConfigured, enabled: true),
                 (CodeCleanupOptions.SortImports, enabled: true),
+                (CodeCleanupOptions.ApplyImplicitExplicitTypePreferences, enabled: false),
```

Comment:
was this needef for some purpose?

Reply:
These are enabled by default at 'None' severity in Tools &rarr; Options. This became 'Refactoring Only' and behaves like all other Hidden-severity diagnostics, which means the fixes will be applied if checked in Code Cleanup.